addons/website_slides/views/slide_channel_partner_views.xml

```diff
@@ -32,7 +35,122 @@
             <field name="name">Attendees</field>
             <field name="res_model">slide.channel.partner</field>
             <field name="view_mode">tree,form</field>
+            <field name="view_id" ref="slide_channel_partner_view_tree"/>
             <field name="search_view_id" ref="website_slides.slide_channel_partner_view_search"/>
         </record>
+
+        <!-- Courses of partner (Stat button on Contact's form view) -->
+        <record id="slide_channel_partner_contact_tree" model="ir.ui.view">
+            <field name="name">slide.channel.partner.contact.view.tree</field>
```

Comment:
Guidelines: model.view.tree.suffix (for example: slide.channel.partner.view.tree.contact).

Update xml id accordingly :)